perun-openapi/openapi.yml

```
@@ -7016,6 +7016,36 @@ paths:
                 user: { type: integer}
                 userExtSource: { $ref: '#/components/schemas/UserExtSource' }
 
+  /urlinjsonout/usersManager/validatePreferredEmailChange:
+    post:
+      tags:
+        - UsersManager
+      operationId: validatePreferredEmailChange
+      summary: Validate new preferred email address. Request to validate is determined based on encrypted parameters sent in email notice by requestPreferredEmailChange() method.
+      parameters:
+        - {name: i, description: "encrypted request parameter", schema: {type: string}, in: query, required: true}
+        - {name: m, description: "encrypted request parameter", schema: {type: string}, in: query, required: true}
+        - {name: u, description: "id of user you want to validate preferred email request", schema: {type: int}, in: query, required: true}
+      responses:
+        '200':
+          $ref: '#/components/responses/StringResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/usersManager/getPendingPreferredEmailChanges:
```

```suggestion
  /urlinjsonout/usersManager/getPendingPreferredEmailChanges:
```
I believe you should use this serializer/deserializer when passing data in url.

---

Sorry, I was wrong, even this should work.